### PR TITLE
GA event recorded if the install page is scrolled past it's initial heig...

### PR DESCRIPTION
**Pull request by cadecairos**
...ht

### CODE REVIEW

File: public/js/pages/install-app.js [hunk 1]
```diff
@@ -28,6 +29,13 @@ require(["jquery", "analytics", "intl-tel-input", "ua-parser-js"], function ($,
     $(".desktop-only").removeClass("desktop-only");
   }
 
+  $(window).scroll(function checkScroll() {
+    if ( window.scrollY >= initialVewportHeight ) {
+      $(window).off("scroll", checkScroll);
+      analytics.event("Scrolled Page");
```
Comment (adamlofting) [inline public/js/pages/install-app.js]: Can we say:
```
analytics.event("Scrolled Past Initial Viewport Height");
```

Comment (cadecairos) [inline public/js/pages/install-app.js]: of course we can!

File: public/js/pages/install-app.js [hunk 2]
```diff
@@ -28,6 +29,13 @@ require(["jquery", "analytics", "intl-tel-input", "ua-parser-js"], function ($,
     $(".desktop-only").removeClass("desktop-only");
   }
 
+  $(window).scroll(function checkScroll() {
+    if ( window.scrollY >= initialVewportHeight ) {
```
Comment (adamlofting) [inline public/js/pages/install-app.js]: Could we set this up so it only fires once?

Comment (cadecairos) [inline public/js/pages/install-app.js]: that's what line 34 does!

Comment (adamlofting) [inline public/js/pages/install-app.js]: :+1:

**Comment by secretrobotron** [comment]
Make sure you beautify this.

**Comment by cadecairos** [comment]
It should be beautified now

**Comment by secretrobotron** [comment]
Oh yeah, disregard.

**Comment by secretrobotron** [comment]
R+!

**Pull request merged**